### DISCUSSION TITLE
Shortcut for illustration

### DISCUSSION THREAD

**Discussion by Npa972** [audacious-media-player/audacious#1431] [Q&A] [answered]
To see album art with a big size i have to select "file" then "parameters" then "plugins" then "general" and finally "illustration".
Is there a way to create a shortcut for this ?
Thanks !
![Capture d’écran_2024-07-14_17-03-58](https://github.com/user-attachments/assets/af441db0-103e-4133-95c9-f4733e2fe3cc)

**Comment by radioactiveman** [comment]
No, there is no shortcut to enable this plugin.

However you could dock the plugin window by dragging it into the main window. And then disable the album art (View -> Show Info Bar Album Art) at the bottom to only show it in one place.

https://github.com/orgs/audacious-media-player/discussions/1400#discussioncomment-9693600

**Comment by Npa972** [comment]
Thanks for your answer !
I like the native interface with small album art and spectrum in the bottom but sometimes i want to see bigger album art (to read the name of the musicians...).
It is not a big deal.

**Comment by radioactiveman** [comment]
Another fast way is to show the Song Info window (shortcut Alt + I). The size of the album art is bigger here. Unfortunately it's not resizable when using the Qt interface.

**Comment by Npa972** [comment]
Gtk3 is not enable in archlinux Audacious 4.4. A bug seems to be open for this.
If album art is resizable with gtk3 I'll give a try with your gtk3 package in Aur.
Thanks again.

**Reply by radioactiveman** [reply]
If your desktop environment supports to configure custom shortcuts, you can also use this command:

```bash
audtool plugin-enable albumart-qt  # or without "-qt" for the GTK plugin
```

**Comment by Npa972** [comment, marked as answer]
Great ! Thank you.
No problem with xfce.
![Capture d’écran_2024-07-14_21-07-37](https://github.com/user-attachments/assets/270bbe33-1c7f-4b6c-8ace-c2c5f674cd77)
